build-llvm34.md

```diff
@@ -50,13 +41,15 @@ If you want to build KLEE with LLVM 2.9, [click here]({{site.baseurl}}/build-llv
 
    Finally, make sure llvm-config is in your path:   
 
-   ```bash
-   $ sudo ln -sf /usr/bin/llvm-config-3.4 /usr/bin/llvm-config
-   ```
-
    That's it for LLVM. If you want to install it manually, please refer to the official [LLVM Getting Started documentation](http://www.llvm.org/docs/GettingStarted.html).<br/><br/>  
 
-3. **Build STP:** KLEE is based on the STP constraint solver, you can find the instructions [here]({{site.baseurl}}/build-stp).
+3. **Install constraint solver(s)**
+
+   KLEE supports multiple different constraint solvers. You must install at least one to build KLEE.
+
+   * [STP](https://github.com/stp/stp) Historically KLEE was built around only this solver so support for this solver is the most stable. For build instructions see [here]({{site.baseurl}}/build-stp). When configuring KLEE in step 9 pass `-DENABLE_SOLVER_STP=ON` to CMake to enable support for STP. If you don't have STP installed but have a built copy you can also pass `-DSTP_DIR=<STP_BUILD_DIR>` to CMake when configuring KLEE where `<STP_BUILD_DIR>` is the absolute path to the STP build directory which contains the file `STPConfig.cmake`.
```

Comment:
I think we should move all these references to Step 9 to Step 9 itself.  I found it hard to remember all this in Step 3.